src/components/Hero/_Hero.scss

```diff
@@ -165,3 +165,22 @@
     }
   }
 }
+
+.hero--tertiary {
+  background-color: var(--ui-gray-xdark);
+
+  .hero__content {
+    @include wrapper;
+
+    color: var(--ui-white);
+    display: flex;
+    flex-flow: column nowrap;
+    padding: var(--space-l) var(--space);
+
+    /* stylelint-disable */
+    .hero__heading {
+      @include space-stack-s;
```

Comment:
Sorry, missed this the first time around.  We need to add some logic/CSS that sets bottom-margin to `space-stack-none` when the `subHeaderText` is not shown.  Either that or wrap the `subHeaderText` in a `<p>` and add top-margin to that element.

Reply:
To keep in alignment with how our CSS is structured for the DS, the first option makes the most sense.

Reply:
I've made some adjustments.  `subHeaderText` needs to be wrapped in a `<p>` tag and with that only bottom margin is set.  Please review

Reply:
Looks good.